docs/developer/chart-repo.md

```diff
@@ -36,11 +36,11 @@ Kubeapps is a Kubernetes-native application. To develop and test Kubeapps compon
 
 ```bash
 cd $MONOCULAR_DIR
-dep ensure
+go mod tidy
```

Comment:
did you need to run this when working with monocular?

Reply:
I think so. Without the packages in $GOPATH/pkg/mod/ downloaded by 'go mod tidy', monocular code cannot compile.

Reply:
when using go modules, any command (like `go build` or `go test`) will download the required modules. `go mod tidy` is used to remove unnecessary dependencies from the `go.mod` file, that's why I found it weird that you needed that.

In any case, the `chart-repo` is meant to be built using it's Dockerfile (and it's built within a container) so running `go mod tidy` won't have any effect.

Reply:
Yes, you're right.  I referred to the monocular doc as below and 'dep ensure' will report error `could not find project Gopkg.toml`.

https://github.com/helm/monocular/blob/master/docs/development.md#developing-chartsvc
![image](https://user-images.githubusercontent.com/10006/61515499-548c9a00-aa35-11e9-91f6-45693582bc5b.png)

Reply:
ah, I see, I don't think it's necessary but we can live it that way for consistency. Thanks!